### PR TITLE
refactor(rest-en): add English document of rest

### PR DESCRIPTION
**Pull request by oucb**

### CODE REVIEW

File: en/rest/error.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
-# 错误处理
+# Error handling
 
-### 错误定义
+### Error definition
 
-HTTP Status Code = 20X 可视为操作成功，状态码大于 300 时错误信息会在响应体里返回。
+HTTP Status Code = 20X can be considered as successful operation. When the status code is greater than 300, the error message will be returned in the response body.。
```
Comment (Zhiw) [inline en/rest/error.md]: 末尾多了一个句号

File: en/rest/error.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ HTTP Status Code = 20X 可视为操作成功，状态码大于 300 时错误信
...
+- errors: Error message which can be empty , and contains the field in which the error occurred and the corresponding error message； 
+- errorCode:Error code to distinguish the type of error.
 
 
 
-### 常见的错误码对照表
+### Common error code comparison table
 
-| 错误码     | 中文含义  | 英文含义 |
+| Error code | Chinese meaning | English Meaning |
```
Comment (Zhiw) [inline en/rest/error.md]: English Meaning -> English meaning

File: en/rest/url.md [hunk 3]
```diff
@@ -1,125 +1,124 @@
...
+}
+```
+In the response information of resource list, **items** is the resource list data, and **meta** is the paging information.
+
+meta definition:
+
+| Name | Type | Description |
+| ------ | --- | ----------- |
+| count  | Integer | Total number of meeting the conditions |
+| limit | Integer | Paging Size |
```
Comment (Zhiw) [inline en/rest/url.md]: Paging Size -> Page size

Comment (oucb) [reply en/rest/url.md]: @Zhiw resolved

**Pull request merged**